resources/frontend/src/App.jsx

```diff
@@ -1,12 +1,116 @@
+import SideBar from "./Components/comman/sidebar/SideBar"
 
-function App() {
+const App = () => {
  
 
   return (
     <>
-       <h1 className="text-3xl font-bold underline">
-      Hello world!
-    </h1>
+       
+<SideBar/>
+<div class="p-4 sm:ml-64">
```

Comment:
This dashboard part also need be separated like sidebar component.

Reply:
ok sir